conf/messages.en

```diff
@@ -317,3 +317,19 @@ correctReturn.select.help.content = If you did not send a return, then it will n
 correctReturn.select.checkYourAnswersLabel = Which return do you need to correct?
 correctReturn.select.error.required = Select which return you need to correct
 correctReturn.select.change.hidden = Select
+
+correctReturn.packagedAsContractPacker.title = Are you reporting liable drink you have packaged as a third party or contract packer at UK sites you operate?
+correctReturn.packagedAsContractPacker.heading = Are you reporting liable drink you have packaged as a third party or contract packer at UK sites you operate?
+correctReturn.packagedAsContractPacker.checkYourAnswersLabel =
```

Comment:
please put in based on returns info

Reply:
in returns this looks like it needs editing too